tests/auto_inject/utils.py

```diff
@@ -37,7 +37,7 @@ def _test_install(self, virtual_machine, profile: bool = False):
             warmup_weblog(vm_context_url)
             request_uuid = make_get_request(vm_context_url)
             logger.info(f"Http request done with uuid: [{request_uuid}] for ip [{vm_ip}]")
-        wait_backend_trace_id(request_uuid, 120.0, profile=profile)
```

Comment:
I've removed this timeout in favor of a retry count to make the exponential backoff logic easier to reason about and more correct. This supplied 120 seconds was for two network requests. In my updated logic, the default is per request. At 6 tries, the max time spent sleeping will be 63 seconds. Additionally, if each request took the full 10 second timeout, the total would be about two minutes per request.